Content.Server/DeltaV/StationEvents/Components/GlimmerMobRuleComponent.cs

```diff
@@ -0,0 +1,13 @@
+using Content.Server.StationEvents.Events;
+
+namespace Content.Server.StationEvents.Components;
+
+[RegisterComponent, Access(typeof(GlimmerMobRule))]
+public sealed partial class GlimmerMobRuleComponent : Component
+{
+    [DataField(required: true)]
+    public EntProtoId MobPrototype = string.Empty;
+}
+
+
+[DataField(required: true)]
```

Comment:
remove :trollface:

Reply:
alright, if that's the last thing, then I'll set this to not be a draft anymore. Thanks for all the help! :D